### ISSUE TITLE
Stuck out of bounds (as a ghost) - north of teldrassil

### ISSUE THREAD

**Issue by matanre** [dalaranwow/dalaran-wow#5697]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

*Edit:*
I noticed I did not write this so my character name is Fanttasma level 11 rogue, just in case a GM can somehow help here. Thx :)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
As a Night-Elf Rogue, I tried to complete the rogue level 10 quest called [Destiny Calls] but accidentally fell off of the tree on which the relevant NPC (called Sethir)  stands on (at around 37, 22 in Teldrassil).

What happened next is pretty aggravating -
Firstly, I fell a VERY long distance and died at around 37, 18
Then, my error was not using the spirit healer but instead I tried to each my corpse as a ghost.
Instead of reaching my corpse I fell too much norther than my corpse (possibly because of the increased speed NEs have as a ghost) so now I'm stuck in no-man's land, practically in no area whatsoever.

I should mention that I have no way to reach a spirit healer or hearth since I am a ghost, hence I am stuck forever.

I tired /reload - no help.
Tried logging out but I can't, so I killed the WoW process and logged in again, still no help.
I also cannot move or jump to try and reach other zones.
I tried "Getting Unstuck": https://github.com/dalaranwow/dalaran-wow/issues/5286
but it appears that I'm somehow "moving" so the game tells me "cannot do this while moving"

*cry*

**Expected behaviour**: Tell us what should happen instead
I am not sure what the expected behavior is but maybe if a char is in an "Unknown" zone then on next login automatically rez him in the closest spirit healer?

**Steps to reproduce the problem**:

1.  Die (preferably whilst in Teldrassil)
2.  Run to 37,22 as a ghost and then fall off the tree
3.  Continue north until you reach "nothing"
4. Get stuck.
5. Profit.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

Adding 3 screenshots:
1. Showing the outer map where my player marker is shown outside of Kalimdor north of Teldrassil
2. Showing where my corpse on the map of Teldrassil and that I have no coords according to the game.
3. The magical view from where my Rogue stands (basically the void)

**Comment by matanre** [comment]
Here are the 3 screenshots:

![oob_01](https://user-images.githubusercontent.com/7031222/53912997-e5ed0800-4062-11e9-901d-86954e76a127.png)
![oob_02](https://user-images.githubusercontent.com/7031222/53913000-e7b6cb80-4062-11e9-94e6-885c5f0bb8f0.png)
![oob_03](https://user-images.githubusercontent.com/7031222/53913005-ea192580-4062-11e9-9947-02106fd2e705.png)

**Comment by MalkaviaDW** [comment]
Hello there,

Write a ticket in-game and a GM will help you out as soon as possible.

Cheers!
